app/client/src/features/events/Questions/QuestionList/QuestionList.tsx

```diff
@@ -114,9 +114,7 @@ export function QuestionList({ className, style, fragmentRef }: Props) {
                             </Grid> */}
                             {isModerator ? (
                                 <InfiniteScroll
-                                    dataLength={
-                                        filteredList.length
-                                    }
+                                    dataLength={filteredList.length}
```

Comment:
We could compress this I think so it's a little cleaner even though it's temprorary code.
```tsx
// TempComponent.tsx
interface TempComponentProps {
children: // idr the type
/* Insert other props necessary for infinite scroll to work */
}

export function TempComponent({ children, ...otherStuff }: TempComponentProps) {
  const [isModerator] = someHook();
  return isModerator ? <InfiniteScroll ...otherStuff>{children}</InfiniteScroll> : <>{children}</>
}
```

With this, it's a little less copy paste.